docs/release_process.md

```diff
@@ -150,27 +170,16 @@ After you push the tag to GitHub, you should also
 We leave the tag on [Stage][stage] for a week so that we (and especially QA)
 can check the tag on GCP infrastructure before we deploy it to production.
 
-On Monday, after the Release Readiness Review:
+On Tuesday, after the Release Readiness review with QA:
 
-1. File an [SRE ticket][sre-form] to deploy the tag to [Prod][prod].
-   - Include a link to the GitHub Release
-2. On the GitHub release, update the summary with a reference to the ticket:
-
-   ```text
-   Planned for release to relay.firefox.com on August 9th, 2022 with SVCSE-1385.
-   ```
-
-On Tuesday:
-
-1. When SRE starts the deploy, "cloudops-jenkins" will send status messages
-   into the #fx-private-relay-eng channel.
-2. When you see `PROMOTE PROD COMPLETE`, do some checks on prod:
+1. Use the [release workflow][release-workflow] to select the tag and deploy
+   to the [Prod][prod] environment by selecting `prod`.
+2. When you see `Application relay... is now running new version of deployments manifests.` in `#fx-private-relay-eng` on Slack, do some checks on prod:
    - Spot-check the site for basic functionality
    - Check [sentry prod project](https://mozilla.sentry.io/releases/?environment=prod) for a spike in any new issues
-   - Check [grafana dashboard](https://earthangel-b40313e5.influxcloud.net/d/qiwPC76Zk/fx-private-relay?orgId=1&refresh=1m&from=now-1h&to=now) for any unexpected spike in ops
-   - (optional) [Run end-to-end tests](https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml) on prod (Note: as of 2023-07-12 these are known-broken. 😢)
+   - Check [grafana dashboard](https://yardstick.mozilla.org/) for any unexpected spike in ops
+   - (optional) [Run end-to-end tests](https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml) on prod
```

Comment:
Are the e2e tests working on prod, or still broken? I'd consider removing this bullet point until are fixed.

Reply:
They should be fixed, per the last few runs! https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml